### ISSUE TITLE
Co accessibility breaks - Cannot allocate vector of size 1032.1 Gb

### ISSUE THREAD

**Issue by markphillippebworth** [aifimmunology/MOCHA#6]
Example:

CD14 Monocyte peak matrix from COVID19 (Early Infection). 
CD14Mono = Data.table output for reproducible peakset on CD14 Monos.

tmp <- makeGRangesFromDataFrame(StringsToGRanges(CD14Mono$tileID))
    mcols(tmp) <- as.data.frame(x[[2]][,-1])
    tmp
co_accessibility(as.data.table(mcols(x)), numCores=40)
#Error: cannot allocate vector of size 1032.1 Gb

I also tried running
co_accessibility(as.data.table(mcols(x)), numCores=10)

However, the process was killed.
